tensorflow_addons/metrics/cohens_kappa_test.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for Cohen's Kappa Metric."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+
+from tensorflow_addons.metrics import CohensKappa
+from tensorflow_addons.utils import test_utils
+
+@test_utils.run_all_in_graph_and_eager_modes
+class CohensKappaTest(tf.test.TestCase):
+  def test_config(self):
+    kp_obj = CohensKappa(name='cohens_kappa')
+    self.assertEqual(kp_obj.name, 'cohens_kappa')
+  
+  def test_kappa_random_score(self):
+    kp_obj1 = CohensKappa(num_classes=5)
+    kp_obj2 = CohensKappa(num_classes=5, weightage='linear')
+    kp_obj3 = CohensKappa(num_classes=5, weightage='quadratic')
+    
+    self.evaluate(tf.compat.v1.variables_initializer(kp_obj1.variables))
+    self.evaluate(tf.compat.v1.variables_initializer(kp_obj2.variables))
+    self.evaluate(tf.compat.v1.variables_initializer(kp_obj3.variables))
+    
+    # random score
+    actuals = np.array([4, 4, 3, 4, 2, 4, 1, 1], dtype=np.int32)
+    preds = np.array([4, 4, 3, 4, 4, 2, 1, 1], dtype=np.int32)
+    actuals = tf.convert_to_tensor(actuals, dtype=tf.int32)
+    preds = tf.convert_to_tensor(preds, dtype=tf.int32)
```

Comment:
Maybe write this like:
`actuals = tf.constant([4, 4, 3, 4, 2, 4, 1, 1], dtype=tf.int32)`
or
`actuals = [4, 4, 3, 4, 2, 4, 1, 1]`.
Seems to be trivial to make a numpy array and convert it to tensor in TF2.0.

Reply:
Sure

Reply:
Fixed